docs/products/kafka/howto/group-requests.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Manage group requests
```

Comment:
Consider adding one more section to this article:
`## View and track requests`
This is at least what the title of the article and its intro/description imply.

Reply:
I have added a section called 'View and track requests.' I fear this is borderline explaining the UI, and the purpose of this page is to view and track requests. The page displays a list of topics, and the UI is pretty self-explanatory. We should avoid explaining the UI as much as possible. By mentioning 'view and track' in the introduction, we set the expectation of what users can do on this screen.